Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -121,119 +121,214 @@ uint64_t AODProducerWorkflowDPL::getTFNumber(uint64_t firstVtxGlBC, int runNumbe
+                        truncateFloatFraction(TMath::Sqrt(track.getSigmaSnp2()), mTrackCovDiag),
+                        truncateFloatFraction(TMath::Sqrt(track.getSigmaTgl2()), mTrackCovDiag),
+                        truncateFloatFraction(TMath::Sqrt(track.getSigma1Pt2()), mTrackCovDiag),
+                        (Char_t)(128. * track.getSigmaZY() / track.getSigmaZ2() / track.getSigmaY2()),
+                        (Char_t)(128. * track.getSigmaSnpY() / track.getSigmaSnp2() / track.getSigmaY2()),
+                        (Char_t)(128. * track.getSigmaSnpZ() / track.getSigmaSnp2() / track.getSigmaZ2()),
+                        (Char_t)(128. * track.getSigmaTglY() / track.getSigmaTgl2() / track.getSigmaY2()),
+                        (Char_t)(128. * track.getSigmaTglZ() / track.getSigmaTgl2() / track.getSigmaZ2()),
+                        (Char_t)(128. * track.getSigmaTglSnp() / track.getSigmaTgl2() / track.getSigmaSnp2()),
+                        (Char_t)(128. * track.getSigma1PtY() / track.getSigma1Pt2() / track.getSigmaY2()),
+                        (Char_t)(128. * track.getSigma1PtZ() / track.getSigma1Pt2() / track.getSigmaZ2()),
+                        (Char_t)(128. * track.getSigma1PtSnp() / track.getSigma1Pt2() / track.getSigmaSnp2()),
+                        (Char_t)(128. * track.getSigma1PtTgl() / track.getSigma1Pt2() / track.getSigmaTgl2()));
+        tracksExtraCursor(0,
+                          truncateFloatFraction(tpcInnerParam, mTrack1Pt),
+                          flags,
+                          itsClusterMap,
+                          tpcNClsFindable,
+                          tpcNClsFindableMinusFound,
+                          tpcNClsFindableMinusCrossedRows,
+                          tpcNClsShared,
+                          trdPattern,
+                          truncateFloatFraction(itsChi2NCl, mTrackCovOffDiag),
+                          truncateFloatFraction(tpcChi2NCl, mTrackCovOffDiag),
+                          truncateFloatFraction(trdChi2, mTrackCovOffDiag),
+                          truncateFloatFraction(tofChi2, mTrackCovOffDiag),
+                          truncateFloatFraction(tpcSignal, mTrackSignal),
+                          truncateFloatFraction(trdSignal, mTrackSignal),
+                          truncateFloatFraction(tofSignal, mTrackSignal),
+                          truncateFloatFraction(length, mTrackSignal),
+                          truncateFloatFraction(tofExpMom, mTrack1Pt),
+                          truncateFloatFraction(trackEtaEMCAL, mTrackPosEMCAL),
+                          truncateFloatFraction(trackPhiEMCAL, mTrackPosEMCAL));
+      }
+      if (trackType == GIndex::Source::TPC) {
+        auto& track = tracksTPC[vTracksIdx[i][0]];
+        collisionID = vCollRefsTPC[vTracksIdx[i][0]];
+        tpcChi2NCl = track.getNClusters() ? track.getChi2() / track.getNClusters() : 0;
+        tpcSignal = track.getdEdx().dEdxTotTPC;
+        tpcNClsFindable = track.getNClusters();
```

Comment:
just for the record: `tpcNClsFindable` is not the same as `track.getNClusters()`, I assume this placeholder at the moment.

Reply:
Yes, this is kind of a placeholder now